components/Footer.tsx

```diff
@@ -165,40 +181,101 @@ const Footer = () => {
           </FooterList>
         </FooterCategory>
         <FooterCategory>
-          <FooterHeader>Technology</FooterHeader>
+          <FooterHeader>
+            <DeveloperBoardIcon style={{ marginLeft: -5, marginRight: 10 }} />
```

Comment:
i dont like the negative margining, it causes problem down the line, only time they should be used is like the `position: absolute`.

can we remove all of them.